crates/sui-graphql-e2e-tests/tests/objects/filter_by_type.exp

```diff
@@ -423,19 +420,16 @@ Response: {
   "data": null,
   "errors": [
     {
-      "message": "Invalid type provided as filter: Invalid struct type: 0x2::coin::Coin<. Got error: unexpected end of tokens",
+      "message": "Failed to parse \"String\": Invalid filter, expected: package[::module[::type[<type_params>]]] or primitive type (occurred while parsing \"ObjectFilter\")",
```

Comment:
look at that error message, beautiful

Reply:
To be honest, this is my least favourite thing about this "hidden scalar" approach -- but that's a problem for another day.